src/persist/src/indexed/encoding.rs

```diff
@@ -765,8 +737,8 @@ mod tests {
     fn encoded_batch_sizes() {
         fn sizes(data: DataGenerator) -> usize {
             let metrics = ColumnarMetrics::disconnected();
-            let updates = data.batches().collect();
-            let updates = BlobTraceUpdates::Row(updates);
+            let updates: Vec<_> = data.batches().collect();
+            let updates = BlobTraceUpdates::Row(ColumnarRecords::concat(&updates, &metrics));
```

Comment:
The only code we have that takes advantage of storing multiple chunks is our test code, here and one other place. This is the smallest change to keep the tests passing, but I'm not sure if it feels good or weird...